src/NDItk/multigroup/AverageFissionEnergyRelease/test/AverageFissionEnergyRelease.test.cpp

```diff
@@ -0,0 +1,106 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_floating_point.hpp>
+using Catch::Matchers::WithinRel;
+
+// what we are testing
+#include "NDItk/multigroup/AverageFissionEnergyRelease.hpp"
+
+// other includes
+
+// convenience typedefs
+using namespace njoy::NDItk;
+using AverageFissionEnergyRelease = multigroup::AverageFissionEnergyRelease;
+
+std::string chunk();
+void verifyChunk( const AverageFissionEnergyRelease& );
+std::string chunkWithInsufficientNumberBoundaries();
```

Comment:
These are from the group boundaries object.

Reply:
I removed the unused declarations in fix/review-part3